rrmngmnt/host.py

```diff
@@ -112,12 +112,20 @@ def executor(self, user=None, pkey=False):
             user = copy.copy(self.root_user)
         return ssh.RemoteExecutor(user, self.ip, use_pkey=pkey)
 
-    def run_command(self, command):
+    def run_command(
+        self, command, input_=None, tcp_timeout=None, io_timeout=None
```

Comment:
what point to add input_=None, tcp_timeout=None, io_timeout=None if not pass it to self.executor().run_cmd